packages/dds/cell/src/test/cell.spec.ts

```diff
@@ -211,6 +212,36 @@ describe("Cell", () => {
                 assert.equal(cell2.get(), pending2, "could not get the set value from pending cell");
             });
         });
+
+        describe("Attributor", () => {
+            beforeEach(() => {
+                containerRuntimeFactory = new MockContainerRuntimeFactory();
+                // Connect the first SharedCell with attribution enabled.
+                cell1 = createConnectedCell("cell1", containerRuntimeFactory, true);
+                // Create a second SharedCell with attribution enabled.
+                cell2 = createConnectedCell("cell2", containerRuntimeFactory, true);
+            });
+
+            it("Retrive proper attribution information", () => {
+                const value1 = "value1";
+                const value2 = "value2";
+                cell1.set(value1);
+                cell2.set(value2);
+                cell2.delete();
+
+                containerRuntimeFactory.processSomeMessages(1);
+
+                // Verify the attributionInfo of SharedCell with 1 pending message
+                assert.notEqual(cell1.getAttributor().getAttributionInfo().user, cell2.getAttributor().getAttributionInfo().user, "could not sync the attribution");
```

Comment:
i like that you have tests verifying the behavior of attribution on pending changes. I think it might be nicer to split the test into several tests which validate more disjoint sets of functionality--this can make it easier when tests are broken to eyeball what might be wrong without looking in too much detail.